src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -44,8 +44,10 @@ public static MvcForm BeginForm(this IHtmlHelper htmlHelper)
         /// </summary>
         /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
         /// <param name="suppressAntiforgery">
-        /// If <c>true</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token. By
-        /// default &lt;form&gt; elements will automatically include an antiforgery token.
+        /// If <c>true</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token.
+        /// If <c>false</c> &lt;form&gt; elements will include an antiforgery token.
+        /// By default, &lt;form&gt; elements will include an antiforgery token only if
+        /// <paramref name="method"/> is not <see cref="FormMethod.Get"/>.
```

Comment:
This PR should have no affect on this API except to change the words after `By default`.